src/views/collections/framework-field-guide/components/signup-form.module.scss

```
@@ -66,6 +66,8 @@
 
 .input:focus {
 	background: var(--slate-700);
+	// TODO(undefined-css-vars): found --outline-focused, code expects --outline_focused
```

Stylelint (17.14.0)

[error] Expected empty line before comment (scss/double-slash-comment-empty-line-before)

(scss/double-slash-comment-empty-line-before)